udts.py

```diff
@@ -603,7 +612,8 @@ def save_config(config_to_save):
     config = { "openfile": "tournament-config.json",
         "use_challonge": False,
         "challonge_id": False,
-        "version": version
+        "version": version,
+        "points_on_win_tie_loss": [1, 0, 0]
```

Comment:
as above, this is a tournament configuration option that should be saved in tournament-config.json